hawkbit-repository/hawkbit-repository-test/src/main/java/org/eclipse/hawkbit/repository/test/util/AbstractIntegrationTest.java

```diff
@@ -94,6 +95,9 @@
     protected TargetManagement targetManagement;
 
     @Autowired
+    protected TenantStatsManagement tenantStatsManagement;
```

Comment:
This field is never used in test, I guess we can just remove it and add it if it's used